internal/compiler-bridge/src/main/scala/xsbt/ExtractAPI.scala

```diff
@@ -215,7 +252,10 @@ class ExtractAPI[GlobalType <: Global](
             xsbti.api.Annotation.of(
               processType(in, a.atp),
               if (a.assocs.isEmpty)
-                Array(xsbti.api.AnnotationArgument.of("", a.args.mkString("(", ",", ")"))) // what else to do with a Tree?
+                Array(
+                  xsbti.api.AnnotationArgument
+                    .of("", ReusableTreePrinter.mkString(a.args, "(", ",", ")"))
+                ) // what else to do with a Tree?
```

Comment:
Hash it! Or at least that's what I do: https://github.com/lampepfl/dotty/blob/d4f1c261f808b185f91bb3e6f3efebd25c39646f/compiler/src/dotty/tools/dotc/sbt/ExtractAPI.scala#L619-L695 :)

Reply:
Nice. It would be preferable for the compiler directly hash much more than it already does, but as you point out the test discovery feature would still need to find annotated members.